pom.xml

```diff
@@ -349,9 +349,14 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>com.github.nscala-time</groupId>
-      <artifactId>nscala-time_${scala.minor.version}</artifactId>
-      <version>2.0.0</version>
+      <groupId>codes.reactive</groupId>
+      <artifactId>scala-time_2.11</artifactId>
```

Comment:
I think what I'll do is remove the dependency on scala-time, as it provides no benefit whatsoever over java.time, and instead is lacking some important methods that java.time does have.